web/main/views.py

```diff
@@ -2154,15 +2154,15 @@ def export(request, node, file_type='docx'):
     """
     if file_type not in ('docx', 'html', 'json'):
         raise Http404
-
+    experimental, aws_lambda, docx_footnotes = [request.GET.get(var) == 'true' if var in request.GET else default
+                                                for var, default in
+                                                [('experimental', settings.FORCE_EXPERIMENTAL_EXPORT),
+                                                ('aws_lambda', settings.FORCE_AWS_LAMBDA_EXPORT),
+                                                ('docx_footnotes', settings.FORCE_DOCX_FOOTNOTES)]]
```

Comment:
nice

Reply:
true, this gives any user access to which ever kind of export they want, if they are clever enough to manipulate the query strings? probably harmless.